CONTRIBUTING.md

```diff
@@ -1,66 +1,67 @@
+See the list of [language codes](https://developer.chrome.com/webstore/i18n#localeTable) we can support.
 
-## Adding a website that is already dark
+## Indicate a Website is Already Dark
 
-If a website is **already dark** and meets the following requirements:
+If a website is **already dark** and meets these requirements:
 - All pages are dark by default regardless of the system's preferred color-scheme.
-- The URL is the actual website address. (No CNAME or URL redirects)
-- The page isn't WIP, sites will change in this phase and could afterwards not meet the requirements.
+- The URL is the actual website address (no CNAME or URL redirects).
+- The page isn't WIP (sites will change in this phase and could afterwards not meet the requirements).
 
-Then, you can **add it to the [dark-sites.config](https://github.com/darkreader/darkreader/blob/master/src/config/dark-sites.config) file**
-*(please, preserve the alphabetical order)*.
+Then, you can add it to this file *(please preserve alphabetic order)*:  
+- **[dark-sites.config](https://github.com/darkreader/darkreader/blob/master/src/config/dark-sites.config)**
 
-## Fixing incorrect inversions
+## Fix incorrect inversions
 
 If certain **parts** of a web page are **incorrectly inverted or styled**,
-you can fix this by specifying the appropriate [**CSS selectors**](https://developer.mozilla.org/docs/Web/CSS/CSS_Selectors) in
-**[dynamic-theme-fixes.config](https://github.com/darkreader/darkreader/blob/master/src/config/dynamic-theme-fixes.config)**
-(for Dynamic Theme mode)
-or **[inversion-fixes.config](https://github.com/darkreader/darkreader/blob/master/src/config/inversion-fixes.config)**
-(for Filter and Filter+ modes) *(__Please, preserve alphabetical order by URL, use short selectors, and preserve code style__)*.
+you can fix this by specifying the appropriate [**CSS selectors**](https://developer.mozilla.org/docs/Web/CSS/CSS_Selectors)
+in the appropriate file *(please preserve alphabetic order by URL, and preserve code style)*:  
+- Fixes for Dynamic Theme mode: **[dynamic-theme-fixes.config](https://github.com/darkreader/darkreader/blob/master/src/config/dynamic-theme-fixes.config)**  
+- Fixes for Filter/Filter+ modes: **[inversion-fixes.config](https://github.com/darkreader/darkreader/blob/master/src/config/inversion-fixes.config)**
 
 Below you can learn how to make a fix for the appropriate mode.
 
-## How to use the Dev Tools
+## How to Use Dev Tools
 
 Dev Tools is designed to **fix minor issues** on a web page
-*(like dark icon on a dark background, removing a bright background, adding a white background to a transparent image, etc.)*.
+*(like a dark icon on a dark background, removing a bright background, adding a white background to a transparent image, etc.)*.
 If the page looks partially dark and bright in **Dynamic mode**, it's considered a bug.
 For **Filter mode**, it is a common practice to invert parts of the page that are already dark.
```

Comment:
```suggestion
For **Filter mode**, it is a common practice to invert elements of the page that are already dark.
```
How about we change the word parts here to elements to help with more consistency in the document? Just like the change that was made on Line 168.

Reply:
> Is this PR able to be merged? Are we waiting on anything else here?

See open suggested changes.

Reply:
Oh alright, I understand now. I didn't know if @Gitoffthelawn had seen all of these open suggestions still pending. Just curious, not trying to be a pest or pushy sounding.

Thank you again. 😃

Reply:
Hi everyone.  Just very busy.  I'll definitely get back to this as time permits.